integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature

```diff
@@ -1,5 +1,16 @@
 Feature: Bridge tests
 
-  Scenario: Bridge fails to be listed as the operation is not implemented yet
+  Scenario: Create a Bridge
     Given authenticate against Manager
-    And the list of Bridge instances is failing with HTTP response code 200
+    When create a new Bridge "mybridge" in cloud provider "aws" and region "us-east-1"
+    Then the list of Bridge instances is containing the Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "provisioning" within 2 minutes
+
+  Scenario: Delete a Bridge
+    Given authenticate against Manager
+    And create a new Bridge "mybridge" in cloud provider "aws" and region "us-east-1"
+    And the list of Bridge instances is containing the Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "provisioning" within 2 minutes
+    When delete the Bridge "mybridge"
+    Then delete the Bridge "mybridge" is failing with HTTP response code 400
```

Comment:
IMHO this test can be commented out so far, as we cannot delete Bridge which is not in a final state.

Reply:
commented the delete scenario for now